app/main.py

```diff
@@ -1,3 +1,19 @@
+import math
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+    course_dict = {}
+
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        return math.ceil(days / 7)
+
+    @classmethod
+    def from_dict(cls, course_dict: dict):
+        return cls(course_dict["name"], course_dict["description"],
+                   cls.days_to_weeks(course_dict["days"]))
```

Comment:
It would be better to use `OnlineCourse.days_to_weeks` here. You will learn about this case in the future videos on OOP